releases/v1.2.12.toml

```diff
@@ -0,0 +1,38 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.11"
+
+pre_release = false
+
+preface = """\
+The twelfth patch release for `containerd` 1.2 includes an updated runc with
+a fix for CVE-2019-19921, an updated version of the opencontainers/selinux
+dependency, which includes a fix for CVE-2019-16884, an updated version of the
+`gopkg.in/yaml.v2` dependency to address CVE-2019-11253, and a Golang update.
+
+### Notable Updates
+
+* Update the runc vendor to v1.0.0-rc10 which includes a mitigation for [CVE-2019-19921](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-19921).
+* Update the opencontainers/selinux which includes a mitigation for [CVE-2019-16884](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-16884).
+* Update Golang runtime to 1.12.15, which includes a fix to the runtime (Go 1.12.14, Go 1.12.15) and and the `net/http` package (Go 1.12.15)
+* A fix to prevent `SIGSEGV` when starting containerd-shim [containerd/containerd#3960](https://github.com/containerd/containerd/pull/3960)
+* Fixes to `exec` [containerd/containerd#3755](https://github.com/containerd/containerd/pull/3755)
+    - Prevent `docker exec` hanging if an earlier `docker exec` left a zombie process
+    - Prevent High system load/CPU utilization with liveness and readiness probes
+    - Prevent Docker healthcheck causing high CPU utilization
+
+* CRI fixes:
+    - Update the `gopkg.in/yaml.v2` vendor to v2.2.8 with a mitigation for [CVE-2019-11253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11253)
```

Comment:
This is pending https://github.com/containerd/cri/pull/1388 to be merged, and containerd/cri to be re-vendored